includes/Exception.php

```diff
@@ -450,10 +450,15 @@ public function report(){
  */
 class UserBlockedError extends ErrorPageError {
 	public function __construct( Block $block ){
-		global $wgLang, $wgRequest;
+		global $wgLang, $wgRequest, $wgUser;
 
 		$blocker = $block->getBlocker();
-		if ( $blocker instanceof User ) { // local user
+		// Wikia change - begin
+		// SUS-288: Hide blocker's username if the block was made by staff/VSTF
+		if ( $block->shouldHideBlockerName() ) {
+			$link = $block->getGroupNameForHiddenBlocker();
+			// Wikia change - end
+		} elseif ( $blocker instanceof User ) { // local user
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)